docs/index.rst

```diff
@@ -242,6 +242,23 @@ you can go to :guilabel:`Settings > Record placements` to enable this feature.
          This will enable you to track the performance of each placement,
          and make adjustments that increase your :abbr:`CTR (click-through rate)`.
 
+Page-specific keywords
+----------------------
+
+EthicalAds allows our advertisers to target ads based on the content of pages.
+This provides value for everyone, giving users more relevent ads while still respecting their privacy.
+
+Publishers can set page-specific keywords dynamically on each page of their site based on the content of the pages.
+For example, if you have a blog post about Kubernetes, you could set tags of `devops` and `kubernetes`.
```

Comment:
@davidfischer we should probably publish a list of known keywords that we target on for users.

Reply:
I thought about that, but let's hold off on it for now. I want to see what people use it for before we publish too much. Maybe just the general categories?

Reply:
I'll ship this, and we can see where to go from here.